### PR TITLE
Ert changes

### PR DESCRIPTION
**Pull request by jroinc**
## About The Pull Request

ERT changes

## Why It's Good For The Game

Rebalances ERT chances to discourage calling before xenomorphs are going to board.

Of the 90% "normal" ERT percentage, half are default friendly (15% PMC, 30% freelancers), while half are hostile (15% ayy, 15% SoM, 15% UPP). However, 83% are human and will thus fight ayys.

60% are a minor boost to the round (freelancer, SoM, UPP)
30% are a major boost to the round (ayy, PMC)
10% are the "joker" ERTs (5% pizza, 5% DS) which either do near-nothing, or will end the round preety fast.

Additionally the PMC SG chance was lowered from 50% to 30%, because 5 SG's spawning in an ERT guaranteed to be friendly to marines is very stronk.

## Changelog
:cl:
balance: Altered ERT spawn chances, lowered PMC S/G spawn chance.
/:cl:

### CODE REVIEW

**Comment by jroinc** [comment]
sonovabitch the larval toxin changes are still in here despite ALREADY being in master REEE.

**Comment by Rohesie** [comment]
No meme ERTs.

**Comment by ArcaneCarrot** [comment]
Praise be to RNGesus

**Comment by spookydonut** [comment]
admin-only erts should never trigger

File: code/modules/reagents/chemistry_reagents/toxin.dm [hunk 1]
```diff
@@ -492,8 +492,6 @@
 	scannable = TRUE
 
 /datum/reagent/toxin/xeno_growthtoxin/on_mob_life(mob/living/L)
-	if(L.getOxyLoss())
```
Comment (spookydonut) [inline code/modules/reagents/chemistry_reagents/toxin.dm]: undocumented change.

File: code/modules/mob/living/carbon/xenomorph/embryo.dm [hunk 2]
```diff
@@ -77,10 +77,10 @@
 	if(CHECK_BITFIELD(affected_mob.restrained_flags, RESTRAINED_XENO_NEST)) //Hosts who are nested in resin nests provide an ideal setting, larva grows faster.
 		counter += 1 + max(0, (0.03 * affected_mob.health)) //Up to +300% faster, depending on the health of the host.
 	else if(stage <= 4)
-		counter++
+		counter += 2 //Free burst time in ~10 min.
 
 	if(affected_mob.reagents.get_reagent_amount(/datum/reagent/toxin/xeno_growthtoxin))
-		counter += 4 //Dramatically accelerates larval growth. You don't want this stuff in your body. Larva hits Stage 5 in just over 3 minutes, assuming the victim has growth toxin for the full duration.
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/xenomorph/embryo.dm]: undocumented change

File: code/modules/mob/living/carbon/xenomorph/embryo.dm [hunk 3]
```diff
@@ -77,10 +77,10 @@
 	if(CHECK_BITFIELD(affected_mob.restrained_flags, RESTRAINED_XENO_NEST)) //Hosts who are nested in resin nests provide an ideal setting, larva grows faster.
 		counter += 1 + max(0, (0.03 * affected_mob.health)) //Up to +300% faster, depending on the health of the host.
 	else if(stage <= 4)
-		counter++
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/xenomorph/embryo.dm]: undocumented change

File: code/modules/reagents/chemistry_reagents/toxin.dm (hunk 1 above)
Comment (jroinc) [reply code/modules/reagents/chemistry_reagents/toxin.dm]: Already in master

File: code/modules/mob/living/carbon/xenomorph/embryo.dm (hunk 2 above)
Comment (jroinc) [reply code/modules/mob/living/carbon/xenomorph/embryo.dm]: Already in master

File: code/modules/mob/living/carbon/xenomorph/embryo.dm (hunk 3 above)
Comment (jroinc) [reply code/modules/mob/living/carbon/xenomorph/embryo.dm]: Already in master

**Comment by jroinc** [comment]
I'm closing this for now because one, I have to rebase to get rid of those accidental changes (not dealing with them showing up in every PR REE) and 2, I might have figured a better way to do this.

~~I'll be baaack~~

**Pull request closed without merging**